scripts/Modules/TTK_config.py

```diff
@@ -0,0 +1,24 @@
+import mkw_translations as trans
+
+# uses R button instead of B button to drift when enabled
+useRbutton = True
+
+# creates up to backupAmount backup files for your race inputs while you are TASing
+# set to 0 to disable backups
+backupAmount = 0
+
+textFilePath = {
+    "player": "MKW_Inputs\\MKW_Player_Inputs_" + trans.course_slot_abbreviation() + ".csv",
+    "ghost": "MKW_Inputs\\MKW_Ghost_Inputs_" + trans.course_slot_abbreviation() + ".csv",
+    "backup": "MKW_Inputs\\Backups\\_backup.csv"
+}
+
+# rksys copy workflow on the saveProgress script:
+# if (createRKSYScopy): read originalRksysPath -> insert created rkg into download slot on license 1 -> save to rksysCopyFilePath
+saveProgress = {
+    "rkgFilePathPlayer": "MKW_Inputs\\MKW_Player_Inputs_" + trans.course_slot_abbreviation() + ".rkg",
+	"rkgFilePathGhost": "MKW_Inputs\\MKW_Ghost_Inputs_" + trans.course_slot_abbreviation() + ".rkg",
+	"createRKSYScopy": False,
+	"rksysCopyFilePath": "MKW_Inputs/rksys.dat",
+	"originalRksysPath": "User/Wii/title/00010004/524d4345/data/rksys.dat"
```

Comment:
Not everyone uses the American version of the game

Reply:
Sorry this was hard-coded in LuigiM's Lua code and given that it isn't even implemented in Python yet I didn't bother to change it. I'll address this by either adding the rksys functionality or taking this snippet out for now.